docs/examples/demoscenario.md

```diff
@@ -0,0 +1,46 @@
+# Using GitHub Connector <!-- omit in toc -->
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Steps](#steps)
+
+## Overview
+
+This purpose of this guide is to show example usage of GitHub Connector, which allows you to to handle GitHub events. With its help, you will create a lamda in Kyma, which reacts to new issues created in the connected repository with a short message accessible in lambda's logs. 
+
+## Prerequisites
+
+- Kyma with GitHub Connector deployed on your cluster (see [installation guide](/chart/githubconnector/README.md)).
+- WebHook configured to deliver payload to the Connector's `/webhook` endpoint.
+
+### Steps
+
+1. Go to the `/chart/demoscenario` directory, where you can find `demoscenario.sh` script.
+
+2. Run the script and supply the name, which has to be the same as the one you used as release name while installing GitHub Connector with helm, and the Namespace in which GitHub Connector application is running.
```

Comment:
```suggestion
2. Run the script that sets the demo scenario up. Pass the name of the Helm release of the GitHub Connector and the Namespace in which you installed it. Run:
```
Do you just pass the Namespace/release name without any flags? Just like: 
`sh demoscenario.sh githubconnector default` ?

Reply:
Yes, for now. It will be changed in the future.